electron/main.ts

```diff
@@ -354,11 +354,10 @@ ipcMain.on('callTool', async (event, { tool, wine, prefix, exe }: Tools) => {
     winePrefix = `'${protonPrefix}/pfx'`
```

Comment:
You will need to remove those quotes from here otherwise you will fix the issue for wine prefixes but will break proton prefixes since we will have double single quotes.